### PR TITLE
Add voxel grid perception module

### PR DESCRIPTION
**Pull request by jslee02**
This PR adds voxel grid perception module that takes ROS point cloud message and updates DART's `VoxelGridShape`.

I requested four reviewers who're relevant to this changes I think. :sweat_smile: Feel free to dismiss if you feel it's not.

**Caveat**
* This code depends on a unreleased DART branch (i.e., [`voxel_grid_api`](https://github.com/dartsim/dart/tree/enhance/voxel_grid_api)). I will release DART with this change once https://github.com/dartsim/dart/pull/1069 is merged.
* This code is compilable but not tested with ROS. It would be great if someone who is accessible with a Intel RealSense can test this.

***

**Before creating a pull request**

- [x] Document new methods and classes
- [x] Format code with `make format`

**Before merging a pull request**

- [x] Set version target by selecting a milestone on the right side
- [x] Summarize this change in `CHANGELOG.md`

### CODE REVIEW

File: include/aikido/perception/VoxelGridModule.hpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  /// Returns const DART VoxelGridShape.
+  std::shared_ptr<const dart::dynamics::VoxelGridShape> getVoxelGridShape()
+      const;
+
+  /// Receives ROS msg and updates the voxel grid with the received ROS msg.
+  ///
+  /// \param[in] sensorOrigin Origin of sensor relative to frame.
+  /// \param[in] inCoordinatesOf Reference frame, determines transform to be
+  /// applied to point cloud and sensor origin.
+  /// \param[in] timestamp Only detections more recent than this timestamp will
```
Comment (brianhou) [inline include/aikido/perception/VoxelGridModule.hpp]: This isn't in the function signature, so the description should either be deleted or the `timestamp` parameter should be added back.

Comment (jslee02) [reply include/aikido/perception/VoxelGridModule.hpp]: Oh, right. It should be `timeout`, and also the docstring should be updated accordingly.

**Comment by brianhou** [comment]
It looks like the DART PR was merged, so should we update the dependency to make this build?

**Comment by jslee02** [comment]
@brianhou Yeah, the PR is merged, but it's not release yet. Let me release 6.6 with the changes of the PR and post it here.

**Comment by jslee02** [comment]
DART 6.6 is released and uploaded to the PPA so Travis CI should be able to install it correctly. :crossed_fingers:

**Comment by jslee02** [comment]
Alright, it passes for Ubuntu. macOS should pass once https://github.com/Homebrew/homebrew-core/pull/30755 is merged.

**Comment by jslee02** [comment]
Now it passes all the CI tests. Review please, reviewers!

**Comment by jslee02** [comment]
Let's merge this and revisit if we find issues! :)

**Pull request merged**